tests/Feature/BookTest.php

```diff
@@ -379,4 +379,47 @@ public function test_getClaimedBooks_success(): void
             });
 
     }
+
+    public function test_getBookSearch_success(): void
+    {
+        Book::factory()->create(['title'=> 'testybook title','claimed' => 0]);
+        Book::factory()->create(['author'=> 'testerson','claimed' => 0]);
+        Book::factory()->create(['blurb'=> 'testybook','claimed' => 0]);
+        Book::factory()->create(['claimed' => 0]);
+
+        $response = $this->getJson('api/books?search=test');
```

Comment:
Worth setting all of the search fields here - you run the risk that one of these test books just happens to have search somewhere as generated by faker, and then you get random test failures